pkg/konfluxgen/fbc-builder.yaml

```diff
@@ -115,12 +115,6 @@ spec:
     name: CHAINS-GIT_COMMIT
     value: $(tasks.clone-repository.results.commit)
   tasks:
-  - name: fbc-related-image-check
-    when:
-    - input: $(params.skip-fbc-related-image-check)
```

Comment:
The "skip-fbc-related-image-check" is now redundant and should be removed too.

Reply:
This part to be removed right? L40-43
- default: "false"
    description: Skip fbc-related-image-check
    name: skip-fbc-related-image-check
    type: string

Reply:
Yeah. And also passing the argument in the PipelineRun.